docs/api/v2/rest-api.md

```diff
@@ -13,14 +13,28 @@ This API is divided in different _namespaces_ which group a set of resources:
 | `/admin` | Privileged access to the internal operations of the node. |
 | `/private` | Provides functionality to encrypt/decrypt `WakuMessage` payloads using either symmetric or asymmetric cryptography. This allows backwards compatibility with Waku v1 nodes. |
 
-The full HTTP REST API documentation can be found here: [TBD]() 
 
 ### API Specification
 
-The HTTP REST API has been designed following the OpenAPI 3.0.3 standard specification format. The OpenAPI specification file can be found here: [TBD]()
+The HTTP REST API has been designed following the OpenAPI 3.0.3 standard specification format. The OpenAPI specification files can be found here:
```

Comment:
For version-controlled documentation we generally use [semantic breaks](https://sembr.org/): generally this just means each sentence (or full clause) in a separate line. These breaks aren't visible in markdown renders, but does help with reviewing and maintaining docs.